hostmap.go

```diff
@@ -133,19 +132,19 @@ func (rs *RelayState) CopyRelayForIdxs() []uint32 {
 	return ret
 }
 
-func (rs *RelayState) RemoveRelay(localIdx uint32) (iputil.VpnIp, bool) {
+func (rs *RelayState) RemoveRelay(localIdx uint32) (netip.Addr, bool) {
 	rs.Lock()
 	defer rs.Unlock()
 	r, ok := rs.relayForByIdx[localIdx]
 	if !ok {
-		return iputil.VpnIp(0), false
+		return netip.Addr{}, false
```

Comment:
Do we want to return unspecified here or invalid?

Reply:
The default/empty value makes sense imo -- also, this function isn't ever called :D